plugins/logs/middleware.go

```diff
@@ -158,5 +175,28 @@ func (l *Logs) recordResponse(request *Request, w *httptest.ResponseRecorder, re
 		return
 	}
 	rec.Response.Body = string(responseBody)
+	if *reqCategory == category.Search {
+		var resBody SearchResponseBody
+		err := json.Unmarshal(responseBody, &resBody)
+		if err != nil {
+			log.Errorln(logTag, "error encountered while parsing the response: ", err)
```

Comment:
@bietkul In L194, you have a `return` present but not over here. It seems like an oversight.

Reply:
@siddharthlatest Let me remove the `return` statement because for `ES6` the `took` value won't be present for `_msearch` and `reactivesearch` and we don't want to fail the log recording.